python/lsst/ts/weatherstation/csc.py

```diff
@@ -118,6 +118,7 @@ async def end_enable(self, id_data):
         self.telemetry_loop_task = asyncio.ensure_future(self.telemetry_loop())
 
         await super().end_enable(id_data)
+        return SalRetCode.CMD_INPROGRESS
```

Comment:
I should probably have been more clear.  The way to send an in progress ack is documented in https://ts-salobj.lsst.io/salobj_cscs.html#writing-a-csc 

In this case you would do:

```
self.cmd_enable.ack_in_progress(data, timeout=self.enable_timeout)
```

you probably want to do this is `begin_enable` though.

Reply:
Thanks for the clarification. I adjusted the code accordingly.